packages/terra-button-group/tests/jest/__snapshots__/ButtonGroup.test.jsx.snap

```diff
@@ -46,7 +46,6 @@ exports[`correctly applies the theme context className 1`] = `
         >
           <button
             aria-disabled={false}
-            aria-pressed={false}
```

Comment:
why are we removing aria-pressed from button group ..?

Reply:
This was getting removed due to the condition in button. Updated button component to add aria-pressed only when "isSelected" is used so that ButtonGroup would continue to receive aria-pressed as before.

aria-pressed is now back in https://github.com/cerner/terra-core/pull/3940/commits/54c4c362b16f27f2bdfc0acb91705ac1e11d9c7e